src/Parser/Hunter/Survival/Modules/Abilities.js

```diff
@@ -87,7 +87,7 @@ class Abilities extends CoreAbilities {
       {
         spell: SPELLS.CALTROPS_TALENT,
         category: Abilities.SPELL_CATEGORIES.ROTATIONAL,
-        cooldown: 15,
+        cooldown: 12,
```

Comment:
I may be wrong but how about this? http://www.wowhead.com/spell=194277/caltrops 
Shows cooldown of 15 seconds? 🤔

Reply:
That is what prompted my fix - but then I was in-game testing something and noticed
![image](https://user-images.githubusercontent.com/29204244/35696099-bba757e6-0786-11e8-8c1a-bf5ccb557391.png)

Reply:
Any leggos or other talents that can affect this?

Reply:
I found it 🤔 updating this and some others post raid, so just leave it up for now 😨